src/app/browse/br-keyword/br-keyword.component.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="br-keyword.component.css">
+</head>
+<body>
+<p>keywordbrowse</p>
+<!-- <<<<<<< Categories-general-layout -->
+<div class="banner">
+	<div class="browse-keyword">
+	    <h1>Search By Keyword: </h1>
+	    <input type="text" id='keyword'>
+	</div> 
+
```

Comment:
Maybe make the search bar a separate component that can be loaded in where needed. This will let you package up the existing logic of a search bar into a component and put it where ever you would like within your app.